firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/Exceptions.kt

```diff
@@ -130,6 +130,9 @@ class ResponseStoppedException(val response: GenerateContentResponse, cause: Thr
 class RequestTimeoutException(message: String, cause: Throwable? = null) :
   GoogleGenerativeAIException(message, cause)
 
+class InvalidLocationException(location: String, cause: Throwable? = null):
```

Comment:
Could we add docs the specify what is considered an invalid location? eg; empty or containing a forward slash.

Reply:
Done. Instead of pin pointing to possible errors, I linked to the page with the list of valid values.